csrc/scheduler/mma_utils.h

```diff
@@ -345,13 +355,20 @@ class CombineMulSum : public IterVisitor {
   //! Goes through the fusion to find mul-sum pairs.
   //! If user sets the caching flags and properties have been previously
   //! computed, then just return cached results.
-  std::vector<MulSumAsMmaProps> generateMulSumCanidates(
+  std::vector<MulSumProperties> generateMulSumCanidates(
       bool use_cached_results = false);
 
   //! Replaces the candidate mul-sum pairs with mma ops.
   //! Please not this will run generateMulSumCandidates again.
   void replaceWithMmaOp();
 
+  //! Check if the fusion has a mul-sum pair
+  //! than can be replaced by a mma op, or has a single mma op.
+  //! This should be run after generaateMulSumCandidates is run.
+  bool isValid() {
```

Comment:
A more descriptive name might be helpful here since "valid" could be interpreted as "this is a valid input for which we can combine mul-sum to produce an MmaOp", which is not true if there is already a single MmaOp.